docs/OpenCLIExamples.rst

```diff
@@ -0,0 +1,346 @@
+==========================
+CLI Open Endpoint Examples
+==========================
+
+The HCA CLI provides users of the Human Cell Atlas (HCA) to access and download data sets from the HCA. This page
+covers how to access the HCA using the HCA command line utility.
+
+*NOTE:* The HCA CLI utility is compatible with Python 3.5+.
+
+
+``hca create-version``
+----------------------
+
+Returns a timestamp in ``DSS_VERSION`` format (e.g., ``1985-04-12T232050.520000Z``), useful for
+versioning bundles or files.
+
+.. note::
+  A version is a timestamp in RFC3339 format that keeps track of the most recent iteration of a
+  bundle or file. A bundle consists of many different data files, and both bundles and files have
```

Comment:
`A bundle consists of many`

->

`A bundle is a collection of many`

Reply:
Fixed in both OpenCLI and OpenAPI documents!